lib/atomic_cache/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module AtomicCache
-  VERSION = "0.5.2.rc1"
+  VERSION = "0.5.3.rc1"
```

Comment:
We chatted over this and Im unclear - maybe a `0.6` version is needed? We arent observing the `wait.klv.present` [increment in the dashboard](https://app.datadoghq.com/dashboard/s58-faw-q7b/memcached-atomic-cache?from_ts=1627676917563&to_ts=1628281717563&live=true)...

Reply:
I talked with Titus about this, and he says that a patch version makes the most sense, as while it is not a bug fix per se, it is not a very noticeable change.